apps/documents/serializers.py

```diff
@@ -106,3 +107,24 @@ def get_presigned_s3(self, obj):
         log_metric('transmission.info', tags={'method': 'documents.generate_presigned_url', 'module': __name__})
 
         return url
+
+    def get_presigned_s3_thumbnail(self, obj):
+        if obj.upload_status != UploadStatus.COMPLETE:
+            return super().get_presigned_s3(obj)
```

Comment:
This is used for generating presigned upload links - the User should not have the ability to upload their own thumbnail.